<?xml version="1.0"?>
<launch>
    <node name="imu" pkg="roomba500" type="imu.py" output="screen" respawn="false">
        <param name="rosout_disable_topics_generation" value="True"/>
        <param name="pollingRateInHertz" value="30"/>
    </node>

    <node name="magnetometer" pkg="roomba500" type="magnetometer.py" output="screen" respawn="false">
        <param name="rosout_disable_topics_generation" value="True"/>
        <param name="pollingRateInHertz" value="10"/>
        <remap from="odom" to="magnetometer/odom"/>
    </node>

    <!-- <node name="imu" pkg="roomba500" type="imu.py" output="screen" respawn="false">
        <param name="rosout_disable_topics_generation" value="True"/>
        <param name="pollingRateInHertz" value="30"/>
        <remap from="imu/data" to="imu/data_raw"/>
    </node>

    <node name="madgwick" pkg="imu_filter_madgwick" type="imu_filter_node" output="screen" respawn="false">
        <param name="use_mag" value="True"/>
        <param name="remove_gravity" value="False"/>
        <param name="publish_tf" value="False"/>
    </node> -->

</launch>
